skt/executable.py

```diff
@@ -106,6 +106,8 @@ def cmd_run(config_set):
 
     save_state(config_set, {'retcode': retcode})
 
+    runner.retcode = retcode
+
```

Comment:
Why do we have to assign the runner's retcode in `executable.py`, when only the runner itself influences it and cares about it? Can't we assign it in `runner.run()` and thus avoid making `executable.py` know about runner's internals?

Reply:
yeah, this was rushed and hackish, sorry